test/controllers/api_export_controller_test.rb

```diff
@@ -0,0 +1,154 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'test_case'
+require 'rspec/mocks/minitest_integration'
+
+class ApiExportControllerTest < ActionDispatch::IntegrationTest
```

Comment:
It is possible to chain parameters ({`case_status: 'confirmed', monitoring: 'true`}) but there are no tests for this case.

Reply:
Added a test in https://github.com/SaraAlert/SaraAlert/pull/823/commits/ead02e3fc1bcc79b2c717085f526b9b2cf59c480